fendermint/vm/actor_interface/src/init.rs

```diff
@@ -20,9 +21,22 @@ define_singleton!(INIT { id: 1, code_id: 2 });
 pub type AddressMap = HashMap<Address, ActorID>;
 
 /// Delegated address of an Ethereum built-in actor.
-fn eth_builtin_deleg_addr(id: ActorID) -> Address {
+///
+/// This is based to what seems to be going on in the `CREATE_EXTERNAL` method
```

Comment:
typo or me not knowing any English?
```suggestion
/// This is based on what seems to be going on in the `CREATE_EXTERNAL` method
```

Reply:
No, you are absolutely right :monocle_face: